public/app/AppWrapper.tsx

```diff
@@ -131,3 +120,18 @@ export class AppWrapper extends Component<AppWrapperProps, AppWrapperState> {
     );
   }
 }
+
+type Props = RouteComponentProps & { route: RouteDescriptor };
+
+function RouteComponent(props: Props) {
```

Comment:
Is this component necessary? Could `GrafanaRoute` use the location hook instead?